src/LanguageServer/Protocol/Features/Diagnostics/EngineV2/DiagnosticIncrementalAnalyzer.StateManager.ProjectStates.cs

```diff
@@ -63,14 +63,6 @@ public IEnumerable<StateSet> GetAllProjectStateSets()
             }
 
             private ProjectAnalyzerStateSets GetOrCreateProjectStateSets(Project project)
-            {
-                // if we can't use cached one, we will create a new analyzer map. which is a bit of waste since
-                // we will create new StateSet for all analyzers. but since this only happens when project analyzer references
-                // are changed, I believe it is acceptable to have a bit of waste for simplicity.
-                return TryGetProjectStateSets(project) ?? CreateProjectStateSets(project);
-            }
-
-            private ProjectAnalyzerStateSets GetOrUpdateProjectStateSets(Project project)
```

Comment:
>GetOrUpdateProjectStateSets [](http://example.com/codeflow?start=45&length=27)

Changed GetOrCreateProjectStateSets to be the same as what GetOrUpdateProjectStateSets was previously.